src/scss/components/_listings.scss

```diff
@@ -0,0 +1,65 @@
+/**
+ * Listing
+ * ===================================
+ * Example – SERP List – Restaurant Information
+ *
+ *
+ */
+
+$listing-default-borderColor: $color-border;
+
+.c-listing {
+    padding-top: 0;
+    padding-bottom: 0;
+}
+
+    .c-listing-item {
+        display: block;
+        position: relative;
+        margin: 0;
+        padding: spacing(x2) 0;
+        border-bottom: 1px solid $listing-default-borderColor;
+
+        .c-listing &:last-child {
+            border: 0;
+        }
+    }
+
+        //when the listing item has a wrapper link descendent, make sure it doesn’t underline everything (only the main title)
+        .c-listing-item-link {
+            display: block;
+            text-decoration: none;
+
+            &:hover,
+            &:active,
+            &:focus {
+                color: currentColor;
+                text-decoration: none;
+            }
+
+            &:hover {
+                text-decoration: none;
+
+                & .c-listing-item-title {
+                    text-decoration: underline;
+                }
+                & .o-btn--primary {
```

Comment:
It's better not to change other components within another component (so that there's no dependency between CSS components).

So this hook would be better declared as `c-listing-btn` and adding that class to the button in the listings.